src/main/java/org/ssh/ipc/system/memory/MemoryMeasurement.java

```diff
@@ -0,0 +1,26 @@
+package org.ssh.ipc.system.memory;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import lombok.Value;
+import org.ssh.ipc.system.Measurement;
+
+/**
+ * @author jeroen.dejong.
+ */
+@Value
+public class MemoryMeasurement implements Measurement {
+  private Long free;
+  private Long used;
+  private Long total;
+  private long timestamp = System.currentTimeMillis();
+
+  @Override
+  public Map<String, Object> toMap() {
+    return ImmutableMap.of(
+        "free", free,
+        "used", used,
+        "total", total
```

Comment:
timestamp might be useful to add, what do you think @romnous ?

Reply:
make it implement `Temporal` then

Reply:
That's not related. it already implements `Temporal` since `Measurement` does. The question is whether `timestamp` should be added as a value in this map.

Reply:
yes